styles/main.css

```diff
@@ -157,3 +157,8 @@ header {
 .back-to-top:hover{
     opacity: 1;
 }
+
+.back-to-top a {
+    position: fixed;
+    bottom: 3.5em;
+}

```

Comment:
Missing rule to fix the position of the hyperlink relative to the browser window